docs/slotfilling.rst

```diff
@@ -298,45 +264,125 @@ The ``submit()`` method is called when the action is run and all slots are fille
     dispatcher.utter_template('utter_submit', tracker)
     return []
 
+Once this is complete, the form is deactivated.
+
+.. note::
+    Some important things to consider:
 
+    - The slot ``requested_slot`` is automatically added to the domain as an
+      unfeaturized slot. If you want to make it featurized, you need to add it
+      to your domain file.
```

Comment:
as `categorical` slot containing all slots that can be required by your different forms